src/MiniProfiler/ProfileProviders/WebRequestProfilerProvider.cs

```diff
@@ -150,7 +150,23 @@ private static void EnsureName(MiniProfiler profiler, HttpRequest request)
         /// </summary>
         public override MiniProfiler GetCurrentProfiler() => Current;
 
+        /// <summary>
+        /// Current head timing.
+        /// </summary>
+        public override Timing CurrentHead
```

Comment:
Perhaps, the name's very unintuitive (e.g. "is this only to profile async things?") but I agree that may be the sane migration approach for users.